sema/ro/creator/__main__.py

```
@@ -0,0 +1,155 @@
+    root = args.root or "."  # default to current folder
+    out = args.out or ("roc-me.yml" if args.make else "ro-crate-metadata.json")
+    root, rocyml = _find_rocyml(root)
+    return Roc(
+        root=root,
+        rocyml=rocyml,
+        out=out,
+        force=args.force,
+    )
+
+
+def _main(*args_list) -> bool:
```

Ruff (0.8.2)

Missing type annotation for `*args_list`

(ANN002)